src/wagtailmarkdown/static/wagtailmarkdown/js/easymde-controller.js

```diff
@@ -0,0 +1,17 @@
+// wagtailmarkdown/static/wagtailmarkdown/js/easymde-controller.js
```

Comment:
@zerolab @katdom13 - I have noticed this a bit in some other 6.0 migrations.

There is no need to put a comment with the filename in the file itself. The documentation just has this to show you an example of where you file might be located.